frontend/javascripts/admin/job/job_list_view.tsx

```diff
@@ -263,20 +274,14 @@ function JobListView() {
             : null}
         </span>
       );
-    } else if (job.type === APIJobType.TRAIN_MODEL) {
+    } else if (job.type === APIJobType.TRAIN_NEURON_MODEL) {
       const numberOfTrainingAnnotations = job.trainingAnnotations.length;
       return (
         <span>
           {`Train model on ${numberOfTrainingAnnotations} ${Utils.pluralize("annotation", numberOfTrainingAnnotations)}. `}
```

Comment:
Any maybe change the wording here a little like `Train **neuron** model on ...` (** should just indicate the change)